client/extensions/woocommerce/woocommerce-services/state/shipping-label/normalize-address.js

```diff
@@ -1,3 +1,8 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
```

Comment:
Let's use the [`Localize`](https://github.com/Automattic/i18n-calypso#localize) HoC instead.

Reply:
This is inside an action though, generating another action by passing a string — is there another way to do it that lets us use a component?

Reply:
Whoops! Nevermind.